merlion/models/utils/nn_modules/enc_dec_autoformer.py

```diff
@@ -0,0 +1,144 @@
+#
```

Comment:
The code structure looks really clean now, thanks for making the changes. While the current structure is okay, my preference would be to put all the code from enc_dec_autoformer.py in autoformer.py (and similarly for the other models), or to make a forecast/autoformer folder. The current structure is fine too, you can take the call on this.

    .
    ├── ...
    ├── forecast/                    
    │   ├── autoformer/          
    |   │   ├── autoformer.py          
    |   │   ├── enc_dec_autoformer.py   
    │   ├── ... 
    └── ...

Reply:
I think Yihao's current structure makes more sense, since it's more aligned with how we handle other models like AutoSARIMA, e.g. we have `autosarima.py` and `models/utils/autosarima_utils.py`.